### ISSUE TITLE
Searching the Manual for an expression didn't work

### ISSUE THREAD

**Issue by alastajj** [Scirra/Construct-bugs#7576]
In the Construct 3 manual I tried searching for "anglediff" so I could get more information on it (wasn't sure if the expression does negative values or not). 

But it gave me no results!

![image](https://github.com/Scirra/Construct-bugs/assets/5744908/bd82b1f3-cb7f-47e9-a523-d7ebe39c626c)

**Comment by WilsonPercival** [comment]
I agree, searching in the documentation works extremely lousy. Sometimes I just can't find what I want.
